qulice-checkstyle/src/main/java/com/qulice/checkstyle/MethodsOrderCheck.java

```diff
@@ -82,7 +82,7 @@ private void checkClass(final DetailAST node) {
     /**
      * Checks order of methods.
      *
-     * @param methods Iterable<DetailAST> methods
+     * @param methods Nodes representing class methods
```

Comment:
@mkordas I've tried but get checkstyle violation - `@param description should start with capital letter`. So, decided to change as you see